backend/app/models/residents.py

```diff
@@ -0,0 +1,39 @@
+from . import db
+from sqlalchemy import inspect, cast, String
+from sqlalchemy.orm.properties import ColumnProperty
+
+class Residents(db.Model):
+    __tablename__ = "residents"
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    initial = db.Column(db.String, nullable=False)
+    room_num = db.Column(db.Integer, nullable=False)
+    date_joined = db.Column(db.Date, nullable=False)
+    date_left = db.Column(db.Date, nullable=True)
+    status = db.Column(
```

Comment:
I feel like it wouldn't hurt to have status, just in case in the future there are new statuses (is that possible 🤔). But I would like to keep it :DDDDDD

Reply:
if there are new statuses, we need to have a feature to allow users to create new statuses which is doable but I don't think its a feature we need so it might be better to remove it for now since it represents the same information as date_left

Reply:
OK FINE.........